.ci/Jenkinsfile

```diff
@@ -138,10 +138,10 @@ pipeline {
                 }
             }
             environment {
-                SONARCLOUD_LOGIN = credentials('SONARCLOUD_LOGIN')
+                SONARCLOUD_LOGIN = credentials('\taaf177ed-92f7-4dd7-a482-b6b407be60ae')
```

Comment:
You have a tab character here. Although it seems to work, I would recommend removing it.

Reply:
I can't see any tab character in IntelliJ